l10n_fr_ecotaxe_sale/tests/test_sale_ecotaxe.py

```diff
@@ -0,0 +1,92 @@
+# © 2021 Akretion (http://www.akretion.com)
+#   @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+class TestsaleEcotaxe(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls):
+        super(TestsaleEcotaxe, cls).setUpClass()
+
+        cls.ecotaxe_classification = cls.env["account.ecotaxe.classification"]
+        cls.ecotaxe_classification1 = cls.ecotaxe_classification.create(
```

Comment:
It's weird, but looking at codecov comments it seems tests aren't running for this module

```
INFO odoo unittest.suite: skipped setUpClass (odoo.addons.l10n_fr_ecotaxe_sale.tests.test_sale_ecotaxe.TestsaleEcotaxe)
```

Looking into this I think this is coming from `AccountTestInvoicingCommon`:

https://github.com/odoo/odoo/blob/c8f7a427b9b9fcdc857a71f99f8448f4d19c9a60/addons/account/tests/common.py#L38

I believe you have to pass the chart of accounts xmlref when calling super, to get this working.

Reply:
@ivantodorovich updated code with argument

Reply:
hmm interesting. it complains about create_sale_partner and i do not see it defined anywhere in previous migrations too... I am on it :)

Reply:
@ivantodorovich  i cannot really run tests locally because they fails on this reference, without that they are skipped:
ValueError: External ID not found in the system: l10n_fr.l10n_fr_pcg_chart_template

i've added dependency to l10n_fr, but no effect. Any idea?

Reply:
I'm not sure if it helps, but maybe try with this: `@tagged('post_install_l10n', 'post_install', '-at_install')`


https://github.com/odoo/odoo/blob/efcb432f4c459b2d2d819c3aed7d63d53fc93809/addons/l10n_fr_fec/tests/test_wizard.py#L14

Reply:
tried it already but no effect. Will keep fixing & pushing here, it doesn't take much time to run the tests and i am not so far from finishing.